api/client/client.go

```diff
@@ -45,11 +44,20 @@ type Remote interface {
 func NewServer(addr string) Remote {
 	addrs := strings.Split(addr, ",")
 
+	for i := range addrs {
+		if !strings.Contains(addrs[i], "http") {
+			addrs[i] = "http://" + addrs[i]
+		}
+	}
+
 	var remote Remote
 
 	if len(addrs) > 1 {
 		remote, _ = NewGroup(addrs, StrategyOrderedList)
 	} else {
+		if !strings.Contains(addr, "http") {
```

Comment:
This should probably only be done at one centralized location. You're normalizing a URL.